frontend/src/pages/modelServing/screens/global/ModelServingGlobal.stories.tsx

```diff
@@ -0,0 +1,148 @@
+import React from 'react';
+
+import { ComponentStory, ComponentMeta } from '@storybook/react';
+import { rest } from 'msw';
+import { within, userEvent } from '@storybook/testing-library';
+import { expect } from '@storybook/jest';
+import { MemoryRouter, Route, Routes } from 'react-router-dom';
+import { mockK8sResourceList } from '~/__mocks__/mockK8sResourceList';
+import { mockProjectK8sResource } from '~/__mocks__/mockProjectK8sResource';
+import { mockServingRuntimeK8sResource } from '~/__mocks__/mockServingRuntimeK8sResource';
+import { mockInferenceServiceK8sResource } from '~/__mocks__/mockInferenceServiceK8sResource';
+import { mockSecretK8sResource } from '~/__mocks__/mockSecretK8sResource';
+import ModelServingContextProvider from '~/pages/modelServing/ModelServingContext';
+import ModelServingGlobal from './ModelServingGlobal';
+
+export default {
+  title: 'ModelServingGlobal',
+  component: ModelServingGlobal,
+  parameters: {
+    msw: {
+      handlers: [
+        rest.get(
+          'api/k8s/apis/serving.kserve.io/v1alpha1/namespaces/test-project/servingruntimes',
+          (req, res, ctx) => res(ctx.json(mockK8sResourceList([mockServingRuntimeK8sResource()]))),
+        ),
+        rest.get(
+          'api/k8s/apis/serving.kserve.io/v1beta1/namespaces/test-project/inferenceservices',
+          (req, res, ctx) =>
+            res(ctx.json(mockK8sResourceList([mockInferenceServiceK8sResource()]))),
+        ),
+        rest.get('/api/k8s/api/v1/namespaces/test-project/secrets', (req, res, ctx) =>
+          res(ctx.json(mockK8sResourceList([mockSecretK8sResource()]))),
+        ),
+        rest.get(
+          '/api/k8s/apis/serving.kserve.io/v1alpha1/namespaces/test-project/servingruntimes/test-model',
+          (req, res, ctx) => res(ctx.json(mockServingRuntimeK8sResource())),
+        ),
+        rest.get('/api/k8s/apis/project.openshift.io/v1/projects', (req, res, ctx) =>
+          res(ctx.json(mockK8sResourceList([mockProjectK8sResource()]))),
+        ),
+      ],
+    },
+  },
+} as ComponentMeta<typeof ModelServingGlobal>;
+
+const Template: ComponentStory<typeof ModelServingGlobal> = (args) => (
+  <div data-testid="story-loaded">
+    <MemoryRouter>
+      <Routes>
+        <Route path="/" element={<ModelServingContextProvider />}>
+          <Route index element={<ModelServingGlobal {...args} />} />
+        </Route>
+      </Routes>
+    </MemoryRouter>
```

Comment:
🤔 Why do we need this?

Reply:
ModelServingGlobal depends on context. Im still looking into better ways to deal with contest in components. The best i got is just including it like this. The route is because modelservingcontextprovider doesnt pass children but instead passes a route exit. 

the provider runs the hooks that fetch the data in the api. so we need to include it in the story so the data is fetched